### PR TITLE
Production Environment (master branch)

### PR DESCRIPTION
**Pull request by lakshmantgld**
## Config.copy.yml
- Removed unwanted fields.

## sagawa/index.js
- removed productionURL

## sagawa/helpers
- removed sagawaEnv check.

## transaction/helpers
- removed squareEnv check.

## serverless.yml
- changed to config.prod.yml.
- added template variable to **stage**.

### CODE REVIEW

File: serverless/db/mongo/models/sagawa/index.js [hunk 1]
```diff
@@ -509,12 +509,10 @@ export default (db) => {
         JWT_SECRET: jwtSecret,
         LAMBDA_ENV: lambdaEnv,
         BASE_URL: baseUrl,
-        PRODUCTION_URL: productionUrl,
       } = process.env;
 
       const token = JWT.sign(payload, jwtSecret);
-      const prodEnv = lambdaEnv === 'production';
-      const trackingLink = `${prodEnv ? `${productionUrl}/#!` : `${baseUrl}/#!`}/tracking?token=${token}`;
+      const trackingLink = `${baseUrl}/tracking?token=${token}`;
```
Comment (TobiahRex) [inline serverless/db/mongo/models/sagawa/index.js]: you need to add `/#!` so it should look like `${baseUrl}/#!/tracking?token=${token}`

**Review by TobiahRex — CHANGES_REQUESTED** [review]
There's a #!/ you left off for the token url.

File: serverless/db/mongo/models/sagawa/index.js (hunk 1 above)
Comment (lakshmantgld) [reply serverless/db/mongo/models/sagawa/index.js]: Nice Catch!!

Comment (lakshmantgld) [reply serverless/db/mongo/models/sagawa/index.js]: Fixed it!!

**Comment by lakshmantgld** [comment]
Introduced different environment in the **front-end:**
- Changed `API_GATEWAY_URL` to `CLIENT_GRAPHQL_URL` and changed the associated code in the `src/`
- Removed the `sanbox` envs and associated code.
- Removed unwanted envs and added new envs in `webpack.envs.js`.
- Updated the `.env_example` according to present env changes.
- Removed the `NODE_ENV` check in the redux store(`src/redux/store/index.js`) and exported based on the environment.

**Comment by TobiahRex** [comment]
Thanks @lakshmantgld 💯

**Pull request merged**